src/mongo/editors/MongoFindOneResultEditor.ts

```diff
@@ -44,4 +44,9 @@ export class MongoFindOneResultEditor implements ICosmosEditor<IMongoDocument> {
         const db = await this._databaseNode.treeItem.getDb();
         return await MongoDocumentTreeItem.update(db.collection(this._collectionName), newDocument);
     }
+
+    public get id(): string {
+        return this._originalDocument._id.toString();
```

Comment:
Shouldn't this be `${this._databaseNode.id}/${this._collectionName}/${this._originalDocument._id.toString()}`